cartographer/mapping_2d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -184,14 +225,16 @@ void ConstraintBuilder::ComputeConstraint(
   const sensor::PointCloud filtered_point_cloud =
       adaptive_voxel_filter_.Filter(*point_cloud);
 
-  // The 'constraint_transform' (i <- j) is computed from:
-  // - a 'filtered_point_cloud' in j,
-  // - the initial guess 'initial_pose' for (map <- j),
-  // - the result 'pose_estimate' of Match() (map <- j).
-  // - the ComputeSubmapPose() (map <- i)
+  // The 'constraint_transform' (submap <- scan) is computed from:
```

Comment:
These were named `i` and `j` to follow `mapping::SparsePoseGraph::Constraint`. Maybe leave both, i.e. `submap i <- scan j`?